### ISSUE TITLE
Sizes of tensors must match except in dimension 2. Expected size 512 but got size 64 for tensor number 1 in the list.

### ISSUE THREAD

**Issue by nomoreadj** [VAST-AI-Research/TripoSR#109]
when im using a 512*512 image,this error occurred.

Error occurred when executing [Comfy3D] TripoSR:

Sizes of tensors must match except in dimension 2. Expected size 512 but got size 64 for tensor number 1 in the list.

File "C:\ComfyUI\execution.py", line 152, in recursive_execute
output_data, output_ui = get_output_data(obj, input_data_all)
^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
File "C:\ComfyUI\execution.py", line 82, in get_output_data
return_values = map_node_over_list(obj, input_data_all, obj.FUNCTION, allow_interrupt=True)
^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
File "C:\ComfyUI\execution.py", line 75, in map_node_over_list
results.append(getattr(obj, func)(**slice_dict(input_data_all, i)))
^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
File "C:\Users\Administrator\AppData\Local\Programs\Python\Python311\Lib\site-packages\torch\utils\_contextlib.py", line 115, in decorate_context
return func(*args, **kwargs)
^^^^^^^^^^^^^^^^^^^^^
File "C:\ComfyUI\custom_nodes\ComfyUI-3D-Pack\nodes.py", line 1884, in run_TSR
image = torch.cat((image, mask), dim=2).detach().cpu().numpy()
^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

and then I changed to a 64*64 image,it worked,but there must be something wrong,right?
